pallets/gear-program/src/lib.rs

```diff
@@ -91,6 +95,10 @@ pub mod pallet {
         WrongWaitList,
     }
 
+    #[pallet::storage]
+    #[pallet::unbounded]
+    pub(crate) type CodeStorage<T: Config> = StorageMap<_, Identity, CodeId, (Code, CodeMetadata)>;
```

Comment:
Why do we need to load the original code every time we access the code storage?

Reply:
Original code should be accessed only when new program is creating or code needs to be re-instrumented